### ISSUE TITLE
Tableaux de bord – Freeze de l'app après un changement de zone de brief

### ISSUE THREAD

**Issue by AdelineCelier** [MTES-MCT/monitorenv#1889]
Comportement observé (et reproduit dans la foulée) : 
- je crée un tableau de bord et y épingle des éléments
- je change la zone du tableau de bord, en cliquant sur le bouton d'édition à côté du tag "polygone dessiné" dans le tableau de bord
- je modifie la zone du tableau de bord sur la carto – a noter que la zone met qques secondes à se mettre à jour une fois que j'ai cliqué sur valider la zone
- je retourne dans le tableau de bord, et lorsque je veux épingler de nouvelles zones, l'app freeze et un bandeau "MonitorEnv ralentit votre navigateur" apparaît. 

![Image](https://github.com/user-attachments/assets/8095c002-a02c-4279-b2f5-32dc27eecf45)

**Comment by claire2212** [comment]
@adeline c'est ok pour toit ce ticket, il est réglé ?

**Comment by AdelineCelier** [comment]
Yes je pense que c'est bon, ce comportement n'est pas réapparu quand j'ai testé les briefs ces derniers jours.
